zipkin-lens/.linguirc

```diff
@@ -6,7 +6,6 @@
     "zh-cn"
   ],
   "sourceLocale": "en",
-  "fallbackLocale": "en",
```

Comment:
`fallbackLocale` is deprecated.

https://lingui.dev/releases/migration-4#configuration-migrations-for-deprecated-options-were-deleted